ark-plonk-book/src/chapter_1.md

```diff
@@ -0,0 +1,373 @@
+
+
+3.  **Verification** $(V_K,C,m,d)$: The algorithm takes the verification key $V_K, C, m$ and $d$ as input and outputs yes or no depending on whether the verification is successful.
+ 
+### Kate Polynomial commitment scheme (PCS)
+
+Plonk uses Polynomial commitments, in particular Kate commitments ( KZG10) to construct fully-succinct zk-SNARK protocol with constant proof size and verification time. KZG10 allows the prover to commit to a set of polynomials and to then show correct evaluation of these polynomials in a set of points with only one small proof. 
+KZG is not the only PC scheme that could be used to construct ZK Proof systems but is the best one in terms of succinctness compared to other schemes like FRI, DARK and IPA where the proof size increases with the increase of the circuit scale. For example, the DARK scheme has logarithmic proof size and linear verification time. In terms of security, the FRI-based ZK-STARKs algorithm provides both quantum-resistance and does not require any trusted setups. The KZG however uses elliptic curve pairing which is not quantum resistant and requires a third-party trusted setup.
+
+We will first provide a formal definition of the original version of Kate commitments  in which the prover commits to a polynomial f and then performs an evaluation opening at a single point. We will then introduce the batched version, which is the one actually used in Plonk. 
+
+#### Terminology:
+
+Let $\phi(x)\in F_p[X]$ be a polynomial of degree $\leq t$ in $F_p[X]$ where $F_p[X]$ is a polynomial ring over the finite field $F_p$ of prime order $p$. Let $G$ be an elliptic curves group of prime order $p$ and such that there exists a symmetric bilinear pairing $e:G\times G\rightarrow G_T$ into the multiplicative group $G_T$ and for which the t-Strong  Diffie-Hellman  (t-SDH)  Assumption holds. Let $g$ be a generator of  $G$. We will be using the multiplicative notation of the DLP assumption explained earlier instead of the additive notation for the EC group.  We will use the following notation: $[x]=g^x​​$
+
+
+
+ 
+ 
+#### Definition: 
+A polynomial  commitment  scheme  consists  of  six  algorithms:\
+**Setup, Commit, Open, VerifyPoly, CreateWitness, and VerifyEval**.
+
+1. **Setup:** This step is run by a trusted or distributed authority and generates a commitment pair of public and private key $(P_K,S_K)$ as follows: $$⟨\delta,[1], [\alpha], [\alpha^2],.........,[\alpha^t]⟩$$  where $\delta=(e,G,G_t)$ is a bilinear pairing group, $⟨[1], [\alpha], [\alpha^2],.........,[\alpha^t]⟩\in G^{t+1}$ and $S_K=\alpha$  such that $\alpha \in F_p$. 
+Note that $S_K$ is not required in the rest of the scheme.
+
+
+2. **Commit:** The prover computes a commitment $C$ which hides the polynomial $\phi(x)$ as follows: $$     C=[\phi(\alpha)] =\prod_{j=0}^{t}([\alpha^j])^{\phi_j}$$
```

Comment:
Shouldn't it be something like:

```
\sum_j \phi_j [\alpha^j]
```

We've been using additive notation for the elliptic curves.